server/src/controllers/invitations.js

```diff
@@ -0,0 +1,75 @@
+import mongoose from 'mongoose';
+import Invitations from '../Schemas/invitations';
+import Users from '../Schemas/users';
+import * as constants from '../constants/failedResponse';
+
+const { ObjectId } = mongoose.Types;
+const getInvitations = async (req, res) => {
+  const invitations = await Invitations.find({ ...req.query })
+    .populate('team')
+    .populate('invitedBy');
+  return res.send(invitations);
+};
+
+const inviteUsers = async (req, res) => {
+  const { invitedUsers, invitedBy, team } = req.body;
+  const invitationObject = {
+    invitedBy: ObjectId(invitedBy),
+    team: ObjectId(team),
+    timestamp: new Date(),
+    status: 'sent',
+  };
+  const invitations = await Promise.all(
+    invitedUsers.map((userId) => {
+      const invitationToSave = {
+        ...invitationObject,
+        invitedUser: ObjectId(userId),
+      };
+      const invitation = new Invitations(invitationToSave);
+      return invitation.save();
+    }),
+  );
+  return res.send(invitations);
+};
+
+const acceptInvitation = async (req, res) => {
+  const invitation = await Invitations.findOneAndUpdate(
+    { _id: ObjectId(req.params.id) },
+    {
+      status: 'accepted',
+    },
+    { new: true },
+  );
+  await Users.findOneAndUpdate(
+    {
+      _id: ObjectId(invitation.invitedUser),
+    },
+    {
+      $push: { teams: ObjectId(invitation.team) },
```

Comment:
Any particular reason that `$` is used here as a prefix? It seems out of convention.

Reply:
Could you recommend another way to do it?

Reply:
Underscores maybe, we've used it elsewhere as well in the codebase.

`_push`

Reply:
I don't think this works.
Could you send me a resource for this?

Reply:
Well, no worries. It's a tiny thing anyway.

Let's spend that time on the bug triage.